tests/utils.py

```diff
@@ -18,6 +18,13 @@
 packageName = 'ga4gh'
 
 
+def zipSameLength(*iterables):
+    length = len(iterables[0])
+    for iterable in iterables[1:]:
+        assert len(iterable) == length
```

Comment:
Maybe be more explicit then, and call it ``zipLists()``? I'm sure it'll lead to confusing bugs if we don't make it clear it's expecting lists. A docstring explaining the intent would be good too.